packages/xsnap/.gitignore

```diff
@@ -1,2 +1,4 @@
 build
 dist
+test/fixture-snap-pool/
```

Comment:
I'm adding a `t.teardown()` to remove these. I guess ignoring them doesn't hurt... or maybe cherry-pick 
b1c59e1?

Reply:
Let's ignore them, too, for fun.